build.sbt

```diff
@@ -5,16 +5,26 @@ import sbtrelease.Version
 name := "hello"
 
 resolvers += Resolver.sonatypeRepo("public")
+resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
 scalaVersion := "2.12.8"
 releaseNextVersion := { ver => Version(ver).map(_.bumpMinor.string).getOrElse("Error") }
 assemblyJarName in assembly := "hello.jar"
 
+addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.0")
```

Comment:
What do the `%%` and `%` operators do?

Reply:
Its just syntactic sugar for adding a version number to the end.